spec/02-integration/09-hybrid_mode/02-start_stop_spec.lua

```diff
@@ -132,3 +132,38 @@ for cluster_protocol, conf in pairs(confs) do
     end
   end)
 end
+
+-- note that lagacy modes still error when CP exits
+describe("when CP exits DP does not error", function()
+  local need_exit = true
+  setup(function()
+    assert(helpers.start_kong({
+      role = "control_plane",
+      prefix = "servroot1",
+      cluster_cert = "spec/fixtures/kong_clustering.crt",
+      cluster_cert_key = "spec/fixtures/kong_clustering.key",
+      cluster_listen = "127.0.0.1:9005",
+    }))
+    assert(helpers.start_kong({
+      role = "data_plane",
+      prefix = "servroot2",
+      cluster_cert = "spec/fixtures/kong_clustering.crt",
+      cluster_cert_key = "spec/fixtures/kong_clustering.key",
+      cluster_control_plane = "127.0.0.1:9005",
+      proxy_listen = "0.0.0.0:9002",
+      database = "off",
+    }))
+  end)
+  teardown(function()
+    if need_exit then
+      helpers.stop_kong("servroot1")
+    end
+    helpers.stop_kong("servroot2")
+  end)
+  it("", function ()
+    helpers.clean_logfile("servroot2/logs/error.log")
+    assert(helpers.stop_kong("servroot1"))
+    need_exit = false
+    assert.logfile("servroot2/logs/error.log").has.no.line("[error]", true)
```

Comment:
This assertion checks for `[error]`, but from what I can tell, most of the error-handling code in wrpc uses `ngx.NOTICE` log level. Is this correct?

Reply:
> 
We can not tell if a wRPC error is really unexpected, so most of the time they are at notice level, and the caller could decide whether it should be an error. In this case, it should not be an error.